src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -54,21 +53,22 @@ boot_comp_pgtbl_expand(size_t n_pte, pgtblcap_t pt, vaddr_t vaddr, struct cobj_h
 	}
 
 	for (i = 0; i < n_pte; i++) {
-		if (!cos_pgtbl_intern_alloc(&boot_info, pt, vaddr, SERVICE_SIZE)) BUG();
+		if (!cos_pgtbl_intern_alloc(boot_info, pt, vaddr, SERVICE_SIZE)) BUG();
 	}
 }
 
 /* Initialize just the captblcap and pgtblcap, due to hack for upcall_fn addr */
 static void
 boot_compinfo_init(spdid_t spdid, captblcap_t *ct, pgtblcap_t *pt, u32_t vaddr)
 {
-	*ct = cos_captbl_alloc(&boot_info);
+	*ct = cos_captbl_alloc(boot_info);
 	assert(*ct);
-	*pt = cos_pgtbl_alloc(&boot_info);
+	*pt = cos_pgtbl_alloc(boot_info);
 	assert(*pt);
 
-	new_comp_cap_info[spdid].compinfo = &new_compinfo[spdid];
-	cos_compinfo_init(new_comp_cap_info[spdid].compinfo, *pt, *ct, 0, (vaddr_t)vaddr, 4, &boot_info);
+	new_comp_cap_info[spdid].defcompinfo = &new_defcompinfo[spdid];
+	new_comp_cap_info[spdid].compinfo =  cos_compinfo_get(&new_defcompinfo[spdid]);
```

Comment:
Is this necessary, or should we just do the `cos_compinfo_get` when we need the `compinfo`?  Not sure.

Spacing is strange here.  Either line up the `=`s or use a single space.